runtime/parachains/src/metrics.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Declaration of the parachain specific runtime metrics.
+
+use polkadot_runtime_metrics::{Counter, CounterVec};
+
+pub use pallet::*;
+
+#[frame_support::pallet]
+pub mod pallet {
+	use super::*;
+
+	#[pallet::pallet]
+	pub struct Pallet<T>(_);
```

Comment:
Why is this a pallet?

Reply:
At the time, it made sense because I wanted to follow the pattern in the parachains runtime (runtime lego pieces). Now that you asked this question, I think it actually doesn't fit the pallet definition since this one provides no runtime logic at all. I'll get rid of the pallet.